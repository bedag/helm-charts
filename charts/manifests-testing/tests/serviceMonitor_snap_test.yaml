suite: ServiceMonitor Snapshot Test
templates:
  - manifests.yaml
tests:
  - it: Full Configured (Snapshot)
    values:
      - ./values/manifests/serviceMonitor.yaml
    asserts:
      - matchSnapshot: {}