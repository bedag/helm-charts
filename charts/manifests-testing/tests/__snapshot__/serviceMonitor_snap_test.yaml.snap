Full Configured (Snapshot):
  1: |
    apiVersion: v1beta1
    kind: ServiceMonitor
    metadata:
      annotations:
        test.annotations/1: "true"
      labels:
        app.kubernetes.io/instance: RELEASE-NAME
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: RELEASE-NAME-manifests-testing
        app.kubernetes.io/version: 0.0.0
        test.label/1: true
      name: RELEASE-NAME-manifests-testing
      namespace: custom-namespace
    spec:
      endpoints:
      - interval: 10s
        path: /
        port: metrics
        scrapeTimeout: 10s
      namespaceSelector:
        matchNames:
        - NAMESPACE
      podTargetLabels:
      - app
      selector:
        matchLabels:
          app.kubernetes.io/instance: RELEASE-NAME
          app.kubernetes.io/name: RELEASE-NAME-manifests-testing
